udata/harvest/actions.py

```diff
@@ -63,7 +63,8 @@ def create_source(name, url, backend,
                   description=None,
                   frequency=DEFAULT_HARVEST_FREQUENCY,
                   owner=None,
-                  organization=None):
+                  organization=None,
+                  config=None):
```

Comment:
Maybe move `)` to a new line and add a trailing comma?

Reply:
:+1: